README.md

```
@@ -1,44 +1,268 @@
 </p>
 
 <h1 align="center">
-He4rt Discord Bot API
+He4rt Developers Platform
 </h1>
+
 <p align="center">
-  <a href="https://discord.gg/he4rt"><img src="https://img.shields.io/github/license/he4rt/he4rt-bot-api?color=6342ED&style=for-the-badge"></a>
-  <a href="https://discord.gg/he4rt"><img src="https://img.shields.io/github/package-json/v/he4rt/he4rt-bot-api?style=for-the-badge&color=6342ED&label="></a>
-<p>
+  <a href="https://discord.gg/he4rt"><img src="https://img.shields.io/github/repo-size/he4rt/heartdevs.com?style=for-the-badge"></a>
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)